lib/KalturaHelpers.php

```diff
@@ -217,6 +217,14 @@ public static function getOption( $name, $default = null ) {
 		}
 	}
 
+	private static function isPluginNetworkActivated() {
+		if ( ! function_exists( 'is_plugin_active_for_network' ) ) {
+			require_once( ABSPATH . '/wp-admin/includes/plugin.php' );
+		}
+
+		return is_plugin_active_for_network( 'all-in-one-video-pack.wordpress/all-in-one-video-pack.php' );
```

Comment:
Let's see if this will still be required based on https://github.com/kaltura/all-in-one-video-pack.wordpress/pull/46/files/02568834afc5f783f596b69cf0c9848c17167c83#r77488230